### PR TITLE
Resolving missing icons for tabs issue #2130

### PR DESCRIPTION
**Pull request by ThisuraThejith**
Resolved the problem of missing icons in the client general tab.

### CODE REVIEW

File: app/views/clients/viewclient.html [hunk 1]
```diff
@@ -140,11 +140,11 @@ <h3 class="client-title">
 				<uib-tab heading="{{'label.heading.general' | translate}}">
                     <br>
                     <div class="container">
-						<div class="row">
+						<div class="row col-md-12 col-sm-12 ng-scope">
```
Comment (ThisuraThejith) [inline app/views/clients/viewclient.html]: Kindly let me know whether we are not suing bootstrap in this branch.

**Comment by mbj36** [comment]
Can you post screenshot of your fix?

**Comment by ThisuraThejith** [comment]
fix

File: app/views/clients/viewclient.html (hunk 1 above)
Comment (mbj36) [inline app/views/clients/viewclient.html]: 1. Why you have used ng-scope here ? 
2. Why you have changed bootstrap ? It shifts in different line with your fix and more button open s on left side which makes difficult to view

**Comment by ThisuraThejith** [comment]
@mbj36 without this the More button was going to the right corner like this. 
fix

Shall we go with div<class="row"> then?

**Comment by mbj36** [comment]
You can go below with more button but it should be visible perfectly when
more button is clicked

**Comment by ThisuraThejith** [comment]
Sure will do.

**Comment by ThisuraThejith** [comment]
Did the check as suggested. Now it looks like this. 

fix

**Comment by mbj36** [comment]
You can refer this guide https://docs.google.com/document/d/1oXQ2mNojyDFkY_x4RBRPaqS-xhpnDE9coQnbmI3Pobw/edit#heading=h.vhgp8hu9moqn

**Comment by ThisuraThejith** [comment]
Thanx @mbj36 for pointing out. Will refix it and send.

**Comment by ThisuraThejith** [comment]
@mbj36 I tried changing the _buttons.css. But can't see the styles.css getting updated accordingly. Am I missing something.

**Comment by mbj36** [comment]
You need to compile sass after changing in scss file. You can compile using `sass styles.scss > ../styles/styles.css` in the styles-dev directory

**Comment by ThisuraThejith** [comment]
Thanks @mbj36. Got it now. I have done the changes as required. Now it looks like below.
fix

**Comment by mbj36** [comment]
Did you pushed your changes ?

**Comment by ThisuraThejith** [comment]
@mbj36 Yes. I have pushed the changes before already. :)

**Comment by mbj36** [comment]
You didn't pushed after compiling or you didn't compiled successfully other wise it would have been change in 3 files.

I am closing this PR, create a new one after you compile successfully

**Comment by ThisuraThejith** [comment]
@mbj36 Oh! Sorry about that! There are changes in .css file also. That was a misunderstanding. Kindly refer https://github.com/openMF/community-app/pull/2149. Thank you very much for your help.

**Pull request closed without merging**